OneBusAway_WatchKit_Extension/data/OBAStopGroupWK.h

```diff
@@ -0,0 +1,28 @@
+//
+//  OBAStopGroup.h
+//  org.onebusaway.iphone
+//
+//  Created by Shannon Young on 4/5/15.
+//  Copyright (c) 2015 OneBusAway. All rights reserved.
+//
+
+#import "OBAModelObjectWK.h"
+
+extern NSString *const OBARequestIdRecentAndBookmarkStopGroups;
+
+@class OBAStopWK;
+
+typedef NS_ENUM (NSInteger, OBAStopGroupType) {
+    OBAStopGroupTypeRecent,
+    OBAStopGroupTypeBookmark,
+    OBAStopGroupTypeNearby
+};
+
+@interface OBAStopGroupWK : OBAModelObjectWK
+
+@property (nonatomic) NSString *name;
```

Comment:
Good point, been a while since I've created my own model objects for a public use project. I will change that.